qiskit_quantuminspire/qi_jobs.py

```diff
@@ -1,16 +1,29 @@
-from datetime import datetime, timezone
+import asyncio
 from typing import Any, List, Union
 
-from compute_api_client import Result as JobResult
+from compute_api_client import ApiClient, PageResult, Result as RawJobResult, ResultsApi
 from qiskit.circuit import QuantumCircuit
 from qiskit.providers import JobV1 as Job
 from qiskit.providers.backend import Backend
 from qiskit.providers.jobstatus import JobStatus
 from qiskit.result.result import Result
 
+from qiskit_quantuminspire.api.client import config
+from qiskit_quantuminspire.api.pagination import PageReader
 from qiskit_quantuminspire.qi_results import QIResult
 
 
+class CircuitExecutionData:
```

Comment:
Shouldn't this be a dataclass? (Or a pydantic model).

Reply:
yes, made the changes